css/styles.css

```diff
@@ -0,0 +1,123 @@
+.background-image {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: -1;
+  background: linear-gradient(rgb(25, 25, 50), rgba(25, 25, 50, 0.5)), linear-gradient(rgba(25, 25, 50, 0.5), rgb(25, 25, 50)), url('../images/final-background-image.png');
+  background-position: center;
+  background-repeat: repeat-x;
+  object-fit: fill;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.justify-right {
+  justify-content: right;
+}
+
+.row-nav-full {
+  margin-top: 10px;
+}
+
+.column-nav-full {
+  width: 90%;
+  margin: auto;
+}
+
+.home-content {
+  position: relative;
+  top: 180px;
+  width: 90%;
+  margin: auto;
+}
+
+#bookmark-icon {
+  color: white;
+  padding: 0 10px;
+  font-size: 1.2rem;
+  cursor: pointer;
+}
+
+.my-watchlist-anchor {
+  cursor: pointer;
+}
+
+#logo {
+  font-size: 2.7rem;
+  color: white;
+}
+
+.movie-journal-margin {
+  margin: 15px 0;
+}
+
+.row-search {
+  padding-top: 40px;
+}
+
+.search-bar {
+  border-radius: 25px;
+  padding: 15px 25px;
+  width: 100%;
+  border-style: none;
+  background-color: rgba(255, 255, 255, 0.224);
+}
+
+.search-bar:focus {
+  outline: none;
+  border: none;
+  box-shadow: 0 0 0 1.5px white;
+  transition: 0.15s;
+}
+
+button {
+  background: transparent;
+  border: none;
+  position: absolute;
+  top: 167px;
+  right: 20px;
```

Comment:
This will come back to bite you. If every button is `absolute` positioned in your app with these arbitrary movements, any time you try to use a button from here on out will be crazy. What are you using this for? Is this something you can do without absolute positioning?